src/assets/styles/other/_material-reduction.scss

```diff
@@ -923,3 +922,7 @@ ix-nested-tree-node > div {
 body .mat-mdc-option {
   min-height: 36px;
 }
+
+body .mat-mdc-checkbox label {
```

Comment:
We shouldn't add generic styles at the end of the pile to fix problems. It might be possible to figure out specific instances of the css that's causing issues and remove that. This will force generic styles anywhere in the UI where you want to use the checkbox. Which means more styles will have to be added to this file.

Reply:
This is default **mdc** checkbox style which is different from our previous implementation, so that's why we need this fix. And there are no css variable to tweak that, so we need to keep it.